tests/terraform/checks/resource/azure/test_MLCCLocalAuthDisabled.py

```diff
@@ -0,0 +1,42 @@
+import os
+import unittest
+
+from checkov.runner_filter import RunnerFilter
+from checkov.terraform.runner import Runner
+from MLCCLoAuthDisabled import check
```

Comment:
```suggestion
from checkov.terraform.checks.resource.azure.MLCCLoAuthDisabled import check
```
I saw a test was failing, I think that is the error 🙂